evaluator/src/test/kotlin/ProjectSourceRuleTest.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2022 EPAM Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.evaluator
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+
+import java.io.File
+
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.utils.test.createSpecTempDir
+
+class ProjectSourceRuleTest : WordSpec({
+    "projectSourceHasFile()" should {
+        "return true if at least one file matches the given glob pattern" {
+            val dir = createSpecTempDir().apply {
+                addFiles(
+                    "README.md",
+                    "module/docs/LICENSE.txt"
+                )
+            }
```

Comment:
Nit: It would actually be shorter and less nested to just write:
```kotlin
val dir = createSpecTempDir()
dir.addFiles(
  "README.md",
   "module/docs/LICENSE.txt"
)
```

Or if `addFiles` would return the directory it could just be:
```kotlin
val dir = createSpecTempDir().addFiles(
  "README.md",
  "module/docs/LICENSE.txt"
)
```

(same for `addDirs`)

Reply:
Are you ok with keeping this as is. In a following PR further tests will be added which call both `addFiles()` and `addDirs()`, so it will be consistent.

If you still prefer to change it, can I do this in a follow up PR, to avoid the rebase hazzle with my following changes?

Reply:
Fleunt API could still be useful but I'll leave that up to you.

Reply:
I'll keep it in mind and probably implement it in a following PR.